### PR TITLE
fix(build): ignore Scoop commits during commit linting

### PR DESCRIPTION
**Pull request by ctrombley**

### CODE REVIEW

File: build/lint.mk [hunk 1]
```diff
@@ -8,7 +8,7 @@ GOFMT        ?= gofmt
 GOIMPORTS    ?= goimports
 
 COMMIT_LINT_CMD   ?= go-gitlint
-COMMIT_LINT_REGEX ?= "(chore|docs|feat|fix|refactor|tests?)(\([^\)]+\))?: .*"
+COMMIT_LINT_REGEX ?= "(Scoop update|(chore|docs|feat|fix|refactor|tests?)(\([^\)]+\))?:) .*"
```
Comment (sanderblue) [inline build/lint.mk:11]: Did we adjust the commit message Scoop makes? @zlesnr's PR was merged and released I believe, so we should be able to use a commit message template for Scoop in our GoReleaser config.

Comment (zlesnr) [reply build/lint.mk:11]: The change to the goreleaser config has been made, but some issue remains.

Comment (ctrombley) [reply build/lint.mk:11]: We did that and somehow we are still getting the old style commits which are now blocking releases.  I know this isn't a "fix-forward" type change that we usually like to do but I wanted to unblock releases while we figure out what's going on.

Comment (sanderblue) [reply build/lint.mk:11]: Totally agreed with this approach for this kind of scenario :)

**Pull request merged**